diff --git a/apps/server/src/core/core.module.ts b/apps/server/src/core/core.module.ts
index 5bd9be1..64901a8 100644
--- a/apps/server/src/core/core.module.ts
+++ b/apps/server/src/core/core.module.ts
@@ -10,6 +10,7 @@ import { WorkspaceModule } from './workspace/workspace.module';
 import { PageModule } from './page/page.module';
 import { AttachmentModule } from './attachment/attachment.module';
 import { CommentModule } from './comment/comment.module';
+import { CommentResolutionModule } from './comment/resolution/comment-resolution.module';
 import { SearchModule } from './search/search.module';
 import { SpaceModule } from './space/space.module';
 import { GroupModule } from './group/group.module';
@@ -34,6 +35,7 @@ import { ClsMiddleware } from 'nestjs-cls';
     PageModule,
     AttachmentModule,
     CommentModule,
+    CommentResolutionModule,
     FavoriteModule,
     SearchModule,
     SpaceModule,
